src/Language/Bitcoin/Script/Descriptors.hs

```diff
@@ -7,6 +7,8 @@ module Language.Bitcoin.Script.Descriptors (
     outputDescriptorAtIndex,
     ScriptDescriptor (..),
     scriptDescriptorAtIndex,
+    ChecksumDescriptor (..),
```

Comment:
Looking at the haddock IMO it flows better if we move these to the "Parsing" section, right at the top.